pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -117,12 +118,17 @@ func (c *Controller) storeDenyConnection(pktIn *ofctrl.PacketIn) error {
 		DestinationPort: packet.DestinationPort,
 		Protocol:        packet.IPProto,
 	}
+	// Make deep copy of IP addresses
 	if packet.IsIPv6 {
-		tuple.SourceAddress = packet.SourceIP.To16()
-		tuple.DestinationAddress = packet.DestinationIP.To16()
+		tuple.SourceAddress = make(net.IP, 16)
```

Comment:
You can also do the following if no need to distinguish between v4 and v6:

```
    dup := make(net.IP, len(ip))
    copy(dup, ip)
```

Reply:
thanks. I re-evaluated the related code. We indeed don't need to distinguish them here.